packages/api/src/router/integration/integration-test-connection.ts

```diff
@@ -28,11 +29,22 @@ export const testConnectionAsync = async (
       source: "form" as const,
     }));
 
-  const decryptedDbSecrets = dbSecrets.map((secret) => ({
-    ...secret,
-    value: decryptSecret(secret.value),
-    source: "db" as const,
-  }));
+  const decryptedDbSecrets = dbSecrets
+    .map((secret) => {
+      try {
+        return {
+          ...secret,
+          value: decryptSecret(secret.value),
+          source: "db" as const,
+        };
+      } catch {
+        logger.warn(
```

Comment:
Do not swallow error

Reply:
What do you mean with this? What would you do and why?

Reply:
I mean that, in JavaScript you can't specify the exception to catch. Hence, any error thrown will trigger the catch block. It would be useful to log the error for troubleshooting. Swallowing is bad because we will never know what happened here

Reply:
Okay so I just add the error as additional parameter?